src/vg/civcraft/mc/citadel/Utility.java

```diff
@@ -418,10 +418,22 @@ public static boolean reinforcementDamaged(Player player, Reinforcement reinforc
         reinforcement.setDurability(durability);
         boolean cancelled = durability > 0;
         if (durability <= 0) {
-            cancelled = reinforcementBroken(null, reinforcement);
+			if (CitadelConfigManager.shouldLogHostileBreaks()) {
```

Comment:
Is there any situation in which 'cancelled = reinforcementBroken(null, reinforcement);' might be false and this might log when it shouldnt? If no, that other check probably shouldnt set cancelled again to make things a bit more clear.